linetools/isgm/abscomponent.py

```diff
@@ -215,9 +223,14 @@ def add_absline(self, absline, tol=0.1*u.arcsec, skip_vel=False):
           Tolerance on matching coordinates
         skip_vel : bool, optional
```

Comment:
I would change `skip_vel=False` to `chk_vel=True` to be consistent with `chk_sep`